flight/PiOS/Common/pios_openlrs.c

```diff
@@ -971,9 +976,14 @@ static void pios_openlrs_tx_frame(pios_openlrs_t openlrs_dev)
 
 	pios_openlrs_do_hop(openlrs_dev);
 
-	int16_t channels[OPENLRS_PPM_NUM_CHANNELS] = {
-		800, 900, 1000, 1100, 1900, 2000, 2040
-	};
+	int16_t channels[OPENLRS_PPM_NUM_CHANNELS];
+
+	for (int i=0; i < OPENLRS_PPM_NUM_CHANNELS; i++) {
```

Comment:
spaces

Reply:
I think this had already gotten fixed by a later commit.